ocdskingfisherprocess/transform/upgrade_1_0_to_1_1.py

```diff
@@ -0,0 +1,83 @@
+from ocdskingfisherprocess.transform.base import BaseTransform
+from ocdskit.upgrade import upgrade_10_11
+import sqlalchemy as sa
+
+
+class Upgrade10To11Transform(BaseTransform):
+    type = 'UPGRADE-1-0-TO-1-1'
+
+    def process(self):
```

Comment:
I'm not sure why `process_all_files` should be a separate method from `process`, as they do the same thing.

Reply:
Yes. fixed.